source/direct_debit/index.html.md.erb

```diff
@@ -0,0 +1,273 @@
+  <td nowrap><code> CANCELLED </code></td> 
+  <td> Cancelled by the payer. </td>
+  </tr>
+  <tr> 
+  <td nowrap><code> ACTIVE </code></td>
+  <td nowrap><code> CANCELLED </code></td>
+  <td> Cancelled by the payer. </td>
+  </tr>
+  </table>
+
+## Payment statuses for Direct Debit
+
+1. `started` - Your service sent an API request to Pay.
+
+2. `pending` - Indicates that payment details have been submitted to the
+payment provider but they are not yet able to confirm whether the payment has
+been successful.
+
+3. The payment reaches one of several finished states:
+
+  * `success` - Mandate set up successfully
+
+  * `failed` - Mandate could not be set up
+
+  * `cancelled` - The user or service cancelled the mandate
+
+  * `error` - Something went wrong with GOV.UK Pay or the underlying Payment Service Provider
+
+There are some differences in this payment lifecycle compared to card
+payments:
+
+1. `created`, `started` and `submitted` are represented as `started`.
+
+2. We have introduced a new, non-terminal state: `pending`.
+
+## Amending an agreement
+
+A payer may wish to amend an agreement, for example to change the bank account
+details, or contact your service to cancel their mandate.
+
+For the first release of Direct Debit functionality we will not provide this
+functionality (to cancel the mandate, bank account switching). Users can
+cancel their mandate at their bank.
+
+## Reference numbers
+
+You can control what appears on the paying user's bank account statement -
+you'll enter this when setting up your account with the DD supplier. For
+example, `<YOUR-SERVICE>`.
+
+Our DD supplier will generate a mandate reference number, which is given to
+the user in the confirmation email. The first part of the reference will be
+generated from the account name that you enter with the DD supplier. This is
+limited to 12 letters.
+
+When creating a mandate for a variable DD, you may supply a reference.
+
+You may want to supply a reference so you can link up an existing case record
+with the Direct Debit transaction. References that your service supplies with
+mandates will not be exposed to the user, but will be available on GOV.UK Pay
+admin tool and through the API.
+
+Users will only receive the mandate reference number (generated by the DD
+supplier). It is a mandatory requirement of the Direct Debit scheme that
+paying users receive their mandate reference number.
+
+## Collecting payments against a mandate
+
+Once a mandate has been set up, you can make an API call to collect a payment
+against the mandate.
+
+You do not need to wait until the mandate is confirmed to send a collection
+request to GOV.UK Pay.
+
+When collecting a payment against a mandate, you must supply a reference and
+description. References that your service supplies with payments will not be
+exposed to the user, but will be available on GOV.UK Pay admin tool and
+through the API.
+
+Every time a payment is requested, we will send an email to the user giving
```

Comment:
we should call out that a payment will take 3 days to be collected if a mandate is confirmed (+2 days to payout to the service's bank account - that could go in a different section) and that the user gets 3 days advance notice 

or payments take +4 days from the time a mandate is submitted

Reply:
Had a go at adding this